tomcat/src/main/java/com/techcourse/controller/LoginController.java

```diff
@@ -0,0 +1,138 @@
+package com.techcourse.controller;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.UUID;
+import com.techcourse.db.InMemoryUserRepository;
+import org.apache.coyote.http.HttpCookie;
+import org.apache.coyote.http.HttpRequest;
+import org.apache.coyote.http.HttpResponse;
+import org.apache.coyote.http11.Http11Processor;
+import org.apache.coyote.session.Session;
+import org.apache.coyote.session.SessionManager;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class LoginController extends AbstractController {
+
+    private static final String RESOURCE_BASE_PATH = "static";
+    private static final Logger log = LoggerFactory.getLogger(Http11Processor.class);
+
+    @Override
+    protected void doGet(HttpRequest request, HttpResponse.HttpResponseBuilder response) throws Exception {
+        String resource = ensureHtmlExtension(request.getPath());
+        String responseBody = loadResourceContent(resource);
+        boolean containsCookie = request.containsHeaders("Cookie");
+        if (containsCookie) {
+            HttpCookie httpCookie = new HttpCookie(request.getHeader("Cookie"));
+            handleCookieRequest(httpCookie, responseBody, response);
+        }
+
+        if (!containsCookie) {
+            buildOkResponse(responseBody, response);
+        }
+    }
+
+    @Override
+    protected void doPost(HttpRequest request, HttpResponse.HttpResponseBuilder response) {
+        String requestBody = request.getRequestBody();
+        String account = getParameter(requestBody, "account");
+        String password = getParameter(requestBody, "password");
+
+        if (findUserByInfo(account, password)) {
+            handleSuccessfulLogin(response, account);
+        }
+
+        if (!findUserByInfo(account, password)) {
+            handleFailedLogin(response);
+        }
```

Comment:
```suggestion
            handleSuccessfulLogin(response, account);
            return;
        }
        handleFailedLogin(response);
```

early return을 해보면 불필요한 조건문을 제거할 수 있을 것 같아요!